linux/src/main.cpp

```diff
@@ -329,6 +329,10 @@ int main(int argc, char* argv[]) {
         }
     }
 
+    finishUrlRequests();
+
+    Tangram::shutdown();
```

Comment:
This shutdown method can be removed I guess. It only needs the stopping of urlWorkers before tileManager gets disposed to not call its tileData callback